### PR TITLE
Adding translations to the API response (rabl)

### PR DESCRIPTION
**Pull request by rbarrera87**
Translation support for the API added

### CODE REVIEW

**Comment by rbarrera87** [comment]
**PAT Evidence**
From Postman
Screen Shot 2020-07-08 at 12 53 50 PM
image

File: app/views/surveyor/export.json.rabl [hunk 1]
```diff
@@ -34,7 +35,7 @@ child :sections => :sections do
       attribute :api_id                 => :uuid
       node(:help_text,              :if => lambda { |a| !a.help_text.blank? }){ |a| a.help_text }
       node(:exclusive,              :if => lambda { |a| a.is_exclusive }){ |a| a.is_exclusive }
-      node(:text, :if => lambda { |a| a.display_type != "image" }){ |a| a.split(a.text, :pre) }
+      node(:text, :if => lambda { |a| a.display_type != "image" }){ |a| a.split(a.translation(I18n.locale)['text'], :pre) }
```
Comment (alexperto) [inline app/views/surveyor/export.json.rabl:38]: not sure if we also would need to add translations to short_text & help_text nodes

**Comment by alexperto** [comment]
looks good, I just left a comment/questions

File: app/views/surveyor/export.json.rabl (hunk 1 above)
Comment (rbarrera87) [reply app/views/surveyor/export.json.rabl:38]: That might be good idea, however I don't think we're using those fields in any survey so far. In addition to that the thing that I don't like is that everytime that we are using `translation` method here, it does one more query :/ so my suggestion is leaving as is right now and test it on stage and see how it behaves there.
What do you think?

**Pull request merged**